### DISCUSSION TITLE
Flsun SR profile

### DISCUSSION THREAD

**Discussion by array81** [OrcaSlicer/OrcaSlicer#1264] [General]
I noticed that OrcaSlicer now has profiles for all FLSun printers except the SR.
Does anyone have a working profile with SR? I tried to create it but I don't know the values to assign to the printer (speed per axis, accelerations, ...).
I've read that someone recommends using that of the V400 but I don't think the speeds and accelerations are excessive.
An "official" base profile for SR would also be desirable.

**Comment by fr3ak2402** [comment]
I've added it to the list and will make it happen this weekend.

**Reply by array81** [reply]
Good. I've created a profile for my SR, it seems to work but I'm not at all sure I've entered the correct speed and acceleration values.

**Reply by fr3ak2402** [reply]
Can you send me all the files of your FLSun SR profile? Then I can expand on it and the profile will be ready faster for all the people who use the OrcaSlicer.

**Reply by FabiodiN** [reply]
are there any news about it?

**Comment by array81** [comment]
These are my files:
[sr.zip](https://github.com/SoftFever/OrcaSlicer/files/11709931/sr.zip)
[sr2.zip](https://github.com/SoftFever/OrcaSlicer/files/11709940/sr2.zip)

**Reply by fr3ak2402** [reply]
I have just finished the profile

**Reply by array81** [reply]
I arrived late. I was away from home, sorry.

**Reply by fr3ak2402** [reply]
No problem, I can simply compare these files later and adjust the speed values. With my profiles, the printer prints much slower...

**Reply by managerman** [reply]
Would love to try these profiles with my SR...How do I go about adding them?  I am using the speeder pad with both my SR's....would these profiles work?

**Reply by fr3ak2402** [reply]
Yes that should also work
You would only have to switch the code from marlin to Klipper and change the start & end code for Klipper.

I could also create a clipper profile of the FLSun Super Racer.

**Reply by managerman** [reply]
A Klipper profile for SR would be fantastic!

**Reply by fr3ak2402** [reply]
Can you send me your start & end code that you use on your SRs?

**Reply by array81** [reply]
In this moment I'm not at home in this moment. I can send them to you in 2/3 hours. In any case you can find them on my SR.ZIP file (on FLSun SR 0.4 nozzle - Copia.json file).

**Reply by fr3ak2402** [reply]
I found it, I was thinking a bit too complicated earlier...

I also didn't notice that you already have Klipper as gcode type. Then I'll just build a Klipper FLSun SR...

It's available as a Marlin and a Klipper machine...

**Reply by array81** [reply]
In fact I don't like it even if it works, it should be modified, I'd like it to make a 90 degree arc at the start and position the nozzle laterally.

**Reply by fr3ak2402** [reply]
So I'm building the Klipper profile from the FLSun SR on top of your files.

The profile of the SR with Marlin is then additional.
Do you have the 90° arc in your profile?

**Reply by array81** [reply]
In my code in this moment I have only a small arc, about 45 degree.
I think it would be good if the arc started on the left and arrived in the central part after completing a 90 degree angle.

**Reply by array81** [reply]
These are my gcode.

START
`;STARTGCODE
M117 Initializing
; Set coordinate modes
G90 ; use absolute coordinates
M83 ; extruder relative mode
; Reset speed and extrusion rates
M200 D0 ; disable volumetric E
M220 S100 ; reset speed
; Set initial warmup temps
M117 Nozzle preheat
M104 S100 ; preheat extruder to no ooze temp
M140 S[first_layer_bed_temperature] ; set bed temp
M190 S[first_layer_bed_temperature] ; wait for bed final temp
M300 S40 P10 ; Bip
; Home
M117 Homing
G28 ; home all with default mesh bed level
; For ABL users put G29 for a leveling request
; Final warmup routine
M117 Final warmup
M104 S[first_layer_temperature] ; set extruder final temp
M109 S[first_layer_temperature] ; wait for extruder final temp
M190 S[first_layer_bed_temperature] ; wait for bed final temp
M300 S440 P200     ; 1st beep for printer ready and allow some time to clean nozzle
M300 S0 P250       ; wait between dual beep
M300 S440 P200     ; 2nd beep for printer ready
G4 S10             ; wait to clean the nozzle
M300 S440 P200     ; 3rd beep for ready to start printing
; Prime line routine
M117 Printing prime line
;M900 K0; Disable Linear Advance (Marlin) for prime line
G92 E0.0; reset extrusion distance
G1 X-54.672 Y-95.203 Z0.3 F4000; go outside print area
G92 E0.0; reset extrusion distance
G1 E2 F1000 ; de-retract and push ooze
G3 X38.904 Y-102.668 I54.672 J95.105 E20.999
G3 X54.671 Y-95.203 I-38.815 J102.373 E5.45800
G92 E0.0
G1 E-5 F3000 ; retract 5mm
G1 X52.931 Y-96.185 F1000 ; wipe
G1 X50.985 Y-97.231 F1000 ; wipe
G1 X49.018 Y-98.238 F1000 ; wipe
G1 X0 Y-109.798 F1000
G1 E4.8 F1500; de-retract
G92 E0.0 ; reset extrusion distance
; Final print adjustments
M117 Preparing to print
;M82 ; extruder absolute mode
M221 S{if layer_height<0.075}100{else}95{endif}
M300 S40 P10 ; chirp
M117 Print [input_filename]; Display: Printing started...`

END
`; printing object ENDGCODE
G92 E0.0 ; prepare to retract
G1 E-6 F3000; retract to avoid stringing
; Anti-stringing end wiggle
{if layer_z < max_print_height}G1 Z{min(layer_z+100, max_print_height)}{endif} F4000 ; Move print head up
G1 X0 Y120 F3000 ; present print
; Reset print setting overrides
G92 E0
M200 D0 ; disable volumetric e
M220 S100 ; reset speed factor to 100%
M221 S100 ; reset extruder factor to 100%
;M900 K0 ; reset linear acceleration(Marlin)
; Shut down printer
M104 S0 ; turn off temperature
M140 S0 ; turn off heatbed
M107 ; turn off fan
M18 S180 ;disable motors after 180s
M300 S40 P10 ; Bip
M117 Print finish.`

LAYER
`;BEFORE_LAYER_CHANGE
;[layer_z]
G92 E0`

This START code draw a 90 degrees arc and I have remove all M200 and M300 code (marlin code) but I need test it. I hope to test it this evening:
`;STARTGCODE
M117 Initializing
; Set coordinate modes
G90 ; use absolute coordinates
M83 ; extruder relative mode
; Reset speed and extrusion rates
M220 S100 ; reset speed
; Set initial warmup temps
M117 Nozzle preheat
M104 S100 ; preheat extruder to no ooze temp
M140 S[first_layer_bed_temperature] ; set bed temp
M190 S[first_layer_bed_temperature] ; wait for bed final temp
; Home
M117 Homing
G28 ; home all with default mesh bed level
; For ABL users put G29 for a leveling request
; Final warmup routine
M117 Final warmup
M104 S[first_layer_temperature] ; set extruder final temp
M109 S[first_layer_temperature] ; wait for extruder final temp
M190 S[first_layer_bed_temperature] ; wait for bed final temp
G4 S10             ; wait to clean the nozzle
; Prime line routine
M117 Printing prime line
;M900 K0; Disable Linear Advance (Marlin) for prime line
G92 E0.0
G1 E-5 F3000 ; retract 5mm
; Lower nozzle and move to start position
G1 Z150
G1 X-125 Y0 Z0.4 F3000
; Extrude about 40 mm by printing a 90 degree arc
G3 X0 Y-125 I125 Z0.3 E40 F2700
; Retract and move nozzle up
G92 E0.0 ; reset extrusion distance
; Final print adjustments
M117 Preparing to print
;M82 ; extruder absolute mode
M221 S{if layer_height<0.075}100{else}95{endif}
M117 Print [input_filename]; Display: Printing started...`

**Reply by array81** [reply]
This START code works and draw 90 degrees arc.
`;STARTGCODE
M117 Initializing
; Set coordinate modes
G90 ; use absolute coordinates
M82 ; extruder absolute mode
; Reset speed and extrusion rates
M220 S100 ; reset speed
; Set initial warmup temps
M117 Nozzle preheat
M104 S100 ; preheat extruder to nozzle temp
M140 S[first_layer_bed_temperature] ; set bed temp
M190 S[first_layer_bed_temperature] ; wait for bed final temp
; Home
M117 Homing
G28 ; home all with default mesh bed level
; Final warmup routine
M117 Final warmup
M104 S[first_layer_temperature] ; set extruder final temp
M109 S[first_layer_temperature] ; wait for extruder final temp
M190 S[first_layer_bed_temperature] ; wait for bed final temp
G4 S10 ; wait to clean the nozzle
; Prime line routine
M117 Printing prime line
;M900 K0; Disable Linear Advance (Marlin) for prime line
G92 E0.0
G1 E-5 F3000 ; retract 5mm
; Lower nozzle and move to start position
G1 Z150
G1 X-125 Y0 Z0.4 F3000
; Extrude about 40 mm by printing a 90 degree arc
G3 X0 Y-125 I125 Z0.3 E40 F2700
; Retract and move nozzle up
G92 E0.0 ; reset extrusion distance
; Final print adjustments
M117 Preparing to print
;M82 ; extruder absolute mode
M221 S{if layer_height<0.075}100{else}95{endif}
M117 Print [input_filename]; Display: Printing started...`

**Reply by Father-Mars** [reply]
Did we ever get the finished profile? I made one but it's mucccch slower than Cura

**Comment by frezeen** [comment]
the sr profile from 1.6.4 beta dont work, print dont start.

i speak about this:
https://github.com/fr3ak2402/OrcaSlicer-Profile-Library/tree/main/profiles/FLSun/machine

**Reply by fr3ak2402** [reply]
Which SR Marlin or Clipper?

I have not finished the profiles yet....

**Comment by frezeen** [comment]
klipper 0.4
whn u start a print  with petg, hotend up to 250, bed stay to 0 and its perm state until u cancel the process. print dont start

**Comment by array81** [comment]
Why the SR profiles is not inclued inside 1.6.4 version?

**Comment by DavideCattaneo97** [comment]
Is the profile for the SR still in the works?

**Comment by platini76** [comment]
I have a big problem using SR with orcaslicer. my print head goes slow and in spurts... hope to use the right word in english.

**Reply by xhenry94** [reply]
me sucede lo mismo

**Comment by nite99a** [comment]
Flsun Super Racer 
working Orca Slicer Profiles without T0 error msg.
And Profile for esun PLA+
[FLSUN.rar](https://www.progipark.com/Flsun/FLSUN.rar)

**Reply by djoe78** [reply]
Hi, I would like to try your profiles but I don't know how to import them into Orca. I use macOS, could you tell me how to do it? thank you so much

**Reply by blueeagle69** [reply]
Hi.
If the Mac version is the same as the Windows version, extract the Rar archive, Open Orca Slicer, and click File/Import/Configs.
Hope this helps.

**Reply by alezz2001** [reply]
I tried to import the files but it always gives me 0 imported configs

![immagine](https://github.com/SoftFever/OrcaSlicer/assets/36056536/14c5ceb8-f3f1-4e73-9c09-f681b918dcbf)

**Reply by blueeagle69** [reply]
Very odd.
I'm using the latest Windows version, and both profiles imported without issue.

Hopefully someone on here has an idea on your issue.
Can't imagine there being a difference in profiles for Mac and Windows.

**Reply by PaoDuke** [reply]
@aleZZ2001 Hi!

I had your same issue.

In Orcaslicer with a generic Marlin settings the same error message that you have show-up.
I solved it first uploading the profile of the FLSUN QQS-PRO and than uploading the config reguarding the FLSUN SR.

Now my question is.. Is this confing for Marlin SR or Klipper SR?
I'm planning to install Klipper on my printer but for now it's stock

**Reply by Silenceikillyou** [reply]
@PaoDuke i have somewhat working(infill flow rate needs to go up but there is no setting or just slow it down) klipper SR profile, where can i drop it?
edit:added separate comment with my repo with full klipper profile even with the stl(copy from cura) so i dont have to stare on blank circle

**Reply by jmichaelwebber** [reply]
It is April 2024, the link to the .rar file is broken: 404 Nicht gefunden

**Comment by RealRogueCmdr** [comment]
> Very odd. I'm using the latest Windows version, and both profiles imported without issue.
> 
> Hopefully someone on here has an idea on your issue. Can't imagine there being a difference in profiles for Mac and Windows.

I did what you said, importing the QQS-PRO then importing your profile, now i get this when I slice 
![image](https://github.com/SoftFever/OrcaSlicer/assets/14956801/101980f3-917e-441c-b100-2717a1786a74)

**Reply by jmichaelwebber** [reply]
I opened the gcode and found the M117 Print as the last line in the start code. I deleted the line and saved the file.  Now the error message does not appear.

**Comment by Silenceikillyou** [comment]
Here is Klipper SR profile: https://github.com/Silenceikillyou/orca-klipper-sr-profile
EDIT: only 0.2 and PLA

**Comment by lucasvsr1** [comment]
Flsun Super-Racer Marlin 
[Flsun SR.zip](https://github.com/SoftFever/OrcaSlicer/files/13551350/Flsun.SR.zip)

**Reply by platini76** [reply]
Luca really you are using orca with sr and marlin without having speed and other problems? For me it is impossible

**Reply by lucasvsr1** [reply]
no

**Reply by cavapanten77** [reply]
Hi platini76, 
Could you tell me the difference between the 3 profiles in your zip file? generic\ 0.4\0.20?
Sorry, but I'm new....
Thank you!!!

**Comment by RealRogueCmdr** [comment]
I have had no issues using orca on FLSun SR.
Sean Cormier
-=RogueCmdr=-



   -
   -

   *“**I will tell you a great secret, Captain. Perhaps the greatest of all
   time. The molecules of your body are the same molecules that make up this
   station, and the nebula outside, that burn inside the stars themselves. We
   are starstuff. We are the universe made manifest, trying to figure itself
   out.**”* Paraphrased – Ambassador Delenn, Babylon 5 (1994)
   -

**Reply by lucasvsr1** [reply]
added your profiles of your sr if you want

**Comment by simonjardine** [comment]
I have a Super Racer, running Klipper.. I tried using the v400 profile then adjusting the bed size and height. No luck...
Why isn't the SR under Flsun? 
I tried the QQS pro, but still no luck..
I think it's down to having the correct Gcode, i managed to get the print started but then the nozzle started cooling?
I was so close, be nice to try Orca :-)
Cheers!!!
Ideally someone who has an SR on klipper could share there Gcode !!

**Reply by Silenceikillyou** [reply]
https://github.com/SoftFever/OrcaSlicer/discussions/1264#discussioncomment-7337253 full pla profile for orca

**Comment by frezeen** [comment]
so, can someone do a recap about sr profiles for stock and klipper??

its incredible how a print like this is not officially supported

**Comment by goldrakes1970** [comment]
I working for this profile

**Comment by Silenceikillyou** [comment]
just updated the profile profile for orca, now with two versions, one for fast well tuned machines running 200mm/s+ and coservative more stable profile if you just want to print and are little lazy like me thats 150mm/s
https://github.com/Silenceikillyou/orca-klipper-sr-profile

**Comment by frezeen** [comment]
Thanks for sharing, ill try It soon.

Are both profiles for stock klipperized SR?

**Reply by Silenceikillyou** [reply]
np, i hope it will work for you, i had some issues with my SR and i think one of the steppers is going to die on me soon(i had to give it max amps to stop skipping at that fast profile and even then it sometimes skipped) and also at least one of the rollers is not in the best shape but its printing good when i limit it to more of a stock settings, its still faster than running marlin plus all the benefits of klipper interface. If yours is in better shape than mine, one of the settings you could definitely lower is the new minimum_cruise_ratio and go a little higher on accel and jerk but i can't test that anymore since i get massive layer shifts because of the stepper if i keep pushing it. I'll keep printing until it dies and then go with a new printer and definely not delta this time.

**Reply by Silenceikillyou** [reply]
"Are both profiles for stock klipperized SR?"
sorry toraly missed that part, i have cutom effector housing, 5015 cooling fans and OMG V2s extruder

**Comment by allenhuffman** [comment]
Just trying Orca for the first time, trying to send the same thing to my SR that I sent to the Bambu Lab A1 Mini to compare. (there was a huge quality different between SR Cura and A1 Mini). Do the profiles shared here supercede the ones in the Orca Slicer I just downloaded tonight? I had no luck with the built in - bed temp was showing 45 and nothing was sticking, and I was about to go tweak but decided to search first and ended up here.

**Comment by Morfdesignco** [comment]
Testing the Orcaslicer profile of the SR I have observed that when printing it takes longer than what the slicing indicates, and when I return to CURA the same problem occurs, I have to reset the printer to factory settings to return to its ideal speed in CURA, however in Orcaslicer still has that error, I have changed the accelerations and has improved a little, but for prints that should last 3 hours can increase up to 5 hours.

**Reply by Lascoose** [reply]
Double check your Marlin version on the printer.  I had the same issue and the Profile comes stock as Marlin (Legacy), my machine was on Marlin 2.
